### PR TITLE
Cleanup/Rewrite `Beatmap/TAG` as `TAG beatmaps`

### PR DESCRIPTION
**Pull request by Nivalyx**
found this fun (but rather outdated) article a while then so i figured i might give a go at it

some notes for consideration :

- the re-titling of the article is intentional to avoid confusion with the "Tags" found in the Song Setup
- yes, [ranked tag beatmaps do exist now](http://osu.ppy.sh/s/1514399)

///

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Review by peppy — DISMISSED** [review]
news sign-off

**Comment by agatemosu** [comment]
You need to update those lines (also add `beatmap/tag: Beatmap/TAG_beatmaps`):
https://github.com/ppy/osu-wiki/blob/a85915d778a96c071cb683e1426691081807d949/wiki/redirect.yaml#L273-L276

**Comment by cl8n** [comment]
I think these are two separate topics that warrant separate articles, or at least sections. the tag co-op modes are what you can play in multiplayer on any map, while tag maps are specifically made for those modes but can be played singleplayer too. seems like the existing links are generally referring to the maps more than the modes.

**Comment by Nivalyx** [comment]
well i mean that's some of the reasons i'm changing the name or the article from `TAG` to `TAG beatmaps` lol

(to clarify as to which "TAG" the page is referring to, as well as to pave way for other "TAG"-related articles, e.g. the ones focusing about the game modes, to be more flexibly written in the future)

File: wiki/Beatmap/TAG_beatmaps/en.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+tags:
+  - multiplayer
+  - co-op
+---
+
+# TAG beatmaps
+
+![An example of a TAG beatmap](img/tag-beatmap-example.png "An example of a TAG beatmap")
+
+*Not to be confused with Tags, which is a [part of a beatmap's metadata](/wiki/Client/Beatmap_editor/Song_setup).*
```
Comment (TicClick) [inline wiki/Beatmap/TAG_beatmaps/en.md:11]:
```suggestion
*Not to be confused with tags, which are a [part of a beatmap's metadata](/wiki/Client/Beatmap_editor/Song_setup).*
```

File: wiki/Beatmap/TAG_beatmaps/en.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+**TAG beatmaps** (or just *TAG*) are beatmaps that are specifically made for [Tag co-op or Tag-team VS mode](/wiki/Client/Interface/Multiplayer#tag-co-op-/-tag-team-vs) in [multiplayer lobbies](/wiki/Client/Interface/Multiplayer). As such, these beatmaps often require cooperative play between two or more players in the lobby in order to be completed.
+
+In these modes, each player is responsible for one [combo chain](/wiki/Beatmapping/Combo), and throughout the song, players alternate between combo chains. For this reason, TAG beatmaps are often designed in such a way that their combo chains are placed very far apart towards each other, making it nearly impossible for the beatmap to be cleared by a single player.
```
Comment (TicClick) [inline wiki/Beatmap/TAG_beatmaps/en.md:15]:
```suggestion
In these modes, each player is responsible for one [combo chain](/wiki/Beatmapping/Combo), and throughout the song the players alternate between combo chains. For this reason, TAG beatmaps are often designed in such a way that their combo chains are placed very far apart from each other, making it nearly impossible for the beatmap to be cleared by a single player.
```

File: wiki/Beatmap/TAG_beatmaps/en.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+TAG [difficulties](/wiki/Beatmap/Difficulty) are most commonly denoted by the use of the term `TAG` in its difficulty name. Most often this `TAG` will be followed by a number, which typically denotes how many players the difficulty is meant for (e.g. `TAG2` for two players or `TAG4` for four players).
```
Comment (TicClick) [inline wiki/Beatmap/TAG_beatmaps/en.md:17]:
```suggestion
TAG [difficulties](/wiki/Beatmap/Difficulty) are most commonly denoted by the use of the term `TAG` in its difficulty name. Most often this `TAG` will be followed by a number, which typically denotes how many players the difficulty is meant for (e.g. `TAG2` for two players, or `TAG4` for four players).
```

File: wiki/Beatmap/TAG_beatmaps/en.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+Due to their nature, TAG difficulties are most commonly unranked, [Approved](/wiki/Beatmap/Category#approved), or [Loved](/wiki/Beatmap/Category#loved). However, on rare occasions, TAG beatmaps that follow the [Ranking Criteria](/wiki/Ranking_criteria) and are deemed suitable for ranking by the [Beatmap Nominators](/wiki/People/Beatmap_Nominators) may found their way into the [Ranked](/wiki/Beatmap/Category#ranked) section.
```
Comment (TicClick) [inline wiki/Beatmap/TAG_beatmaps/en.md:19]:
```suggestion
Due to their nature, TAG difficulties are most commonly unranked, [Approved](/wiki/Beatmap/Category#approved), or [Loved](/wiki/Beatmap/Category#loved). However, on rare occasions, TAG beatmaps that follow the [ranking criteria](/wiki/Ranking_criteria) and are deemed suitable for ranking by the [Beatmap Nominators](/wiki/People/Beatmap_Nominators) may found their way into the [Ranked](/wiki/Beatmap/Category#ranked) section.
```

**Review by TicClick — DISMISSED** [review]
looks good to me otherwise -- I recommend briefly consulting with @Walavouchey w.r.t. my proposed corrections

**Review by peppy — APPROVED** [review]
Seems fine to me.

File: wiki/Beatmap/TAG_beatmaps/en.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+In these modes, each player is responsible for one [combo chain](/wiki/Beatmapping/Combo), and throughout the song the players alternate between combo chains. For this reason, TAG beatmaps are often designed in such a way that their combo chains are placed very far apart from each other, making it nearly impossible for the beatmap to be cleared by a single player.
+
+TAG [difficulties](/wiki/Beatmap/Difficulty) are most commonly denoted by the use of the term `TAG` in its difficulty name. Most often this `TAG` will be followed by a number, which typically denotes how many players the difficulty is meant for (e.g. `TAG2` for two players, or `TAG4` for four players).
+
+Due to their nature, TAG difficulties are most commonly unranked, [Approved](/wiki/Beatmap/Category#approved), or [Loved](/wiki/Beatmap/Category#loved). However, on rare occasions, TAG beatmaps that follow the [ranking criteria](/wiki/Ranking_criteria) and are deemed suitable for ranking by the [Beatmap Nominators](/wiki/People/Beatmap_Nominators) may found their way into the [Ranked](/wiki/Beatmap/Category#ranked) section.
```
Comment (Walavouchey) [inline wiki/Beatmap/TAG_beatmaps/en.md:19]:
```suggestion
Due to their nature, TAG difficulties are most commonly unranked, [Approved](/wiki/Beatmap/Category#approved), or [Loved](/wiki/Beatmap/Category#loved). However, on rare occasions, TAG beatmaps that follow the [ranking criteria](/wiki/Ranking_criteria) and are deemed suitable for ranking by the [Beatmap Nominators](/wiki/People/Beatmap_Nominators) may find their way into the [Ranked](/wiki/Beatmap/Category#ranked) section.
```

**Pull request merged**